locales.software/1038-hu.cfos.txt

```diff
@@ -91,14 +91,14 @@ pnet_conn_local_ip = "Helyi IP cím %?";
 pnet_conn_public_ip = "Nyílvános IP cím %?";
 pnet_pubdir_header = "Nyílvános könyvtár";
 pnet_pubdir_subheader = "Az a könyvtár, amelyik tartalmazza a letölthető fájlokat.";
-pnet_pubdir_info = "Kérlek, add meg azt a mappa helyét, ahol azok a fájlok találhatók, amelyeket elérhetővé szeretnél tenni a letöltéshez.";
+pnet_pubdir_info = "Kérlek, add meg annak a mappának a helyét, ahol azok a fájlok találhatók, amelyeket elérhetővé szeretnél tenni a letöltéshez.";
 smtp_header = "SMTP-szerver";
 smtp_subheader = "SMTP-szerver beállítása.";
 dynip_header = "Dinamikus IP-szerver";
 dynip_subheader = "Dinamikus IP-szerver beállítása";
 dlg_start_now_as_current_user = "Futtasd a %p-t, a jelenlegi felhasználóval";
 dlg_autostart = "%p indítása a Windows-zal együtt";
-dlg_autostart_service = "Telepítsd a %p, mint rendszer folyamat és bekapcsolásnál induljon el a Windows-al együtt";
+dlg_autostart_service = "Telepítés (%p) mint rendszer folyamat és bekapcsolásnál induljon el a Windows-zal együtt";
```

Comment:
Particularly `p%` placeholder always upholds English program name.

Just look into section name to determinate which program name will be there.

For example, here we have:

```
/ ----------------------------------------------------------------------
/
/  6link / broadband connect / pnet
/
/ ----------------------------------------------------------------------
```

So we can see that `%p` will be replaced with `cFos IPv6 Link`, `cFos Broadband Connect` or `cFos Personal Net` (those apps share those strings), as per http://www.cfos.de/en/index.htm products.

To be more exact, in that string will be used `cFos Personal Net`.

Will it help to write article before them properly?